### ISSUE TITLE
Brew wants Xcode 14.1 but the latest available in App Store is 14.0.1

### ISSUE THREAD

**Issue by Helloyunho** [Homebrew/brew#14064]
### `brew config` output

```shell
HOMEBREW_VERSION: 3.6.7
ORIGIN: https://github.com/Homebrew/brew
HEAD: 6a7eac25e167a1eb2d49e13c8cc530a3188af995
Last commit: 7 days ago
Core tap ORIGIN: https://github.com/Homebrew/homebrew-core
Core tap HEAD: 9345e061435f18a91437cc5a3db34a90acbb9f1b
Core tap last commit: 5 minutes ago
Core tap branch: master
HOMEBREW_PREFIX: /opt/homebrew
HOMEBREW_CASK_OPTS: []
HOMEBREW_EDITOR: /usr/bin/vim
HOMEBREW_MAKE_JOBS: 10
Homebrew Ruby: 2.6.10 => /System/Library/Frameworks/Ruby.framework/Versions/2.6/usr/bin/ruby
CPU: 10-core 64-bit arm_firestorm_icestorm
Clang: 14.0.0 build 1400
Git: 2.37.0 => /Applications/Xcode.app/Contents/Developer/usr/bin/git
Curl: 7.84.0 => /usr/bin/curl
macOS: 13.0-arm64
CLT: 14.0.0.0.1.1661618636
Xcode: 14.0.1
Rosetta 2: false
```

### `brew doctor` output

```shell
# NOTE: Ignore "unexpected (something)" since it's for my lib. Also, I tried what brew said to update.

Warning: A newer Command Line Tools release is available.
Update them from Software Update in System Preferences or run:
  softwareupdate --all --install --force

If that doesn't show you any updates, run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/all/.
You should download the Command Line Tools for Xcode 14.1.


Warning: Some installed formulae are deprecated or disabled.
You should find replacements for the following formulae:
  natalie

Warning: Unbrewed dylibs were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected dylibs:
  /usr/local/lib/libgit2.0.28.4.dylib
  /usr/local/lib/ss_conn_lib2.dylib

Warning: Unbrewed header files were found in /usr/local/include.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected header files:
  /usr/local/include/git2.h
  /usr/local/include/git2/annotated_commit.h
  /usr/local/include/git2/apply.h
# ...

Warning: Unbrewed '.pc' files were found in /usr/local/lib/pkgconfig.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected '.pc' files:
  /usr/local/lib/pkgconfig/libgit2.pc

Warning: Unbrewed static libraries were found in /usr/local/lib.
If you didn't put them there on purpose they could cause problems when
building Homebrew formulae, and may need to be deleted.

Unexpected static libraries:
  /usr/local/lib/libgit2.a

Warning: You have unlinked kegs in your Cellar.
Leaving kegs unlinked can lead to build-trouble and cause formulae that depend on
those kegs to fail to run properly once built. Run `brew link` on these:
  numpy
  pillow
  six
  wireshark

Warning: Your Command Line Tools (CLT) does not support macOS 13.
It is either outdated or was modified.
Please update your Command Line Tools (CLT) or delete it if no updates are available.
Update them from Software Update in System Preferences or run:
  softwareupdate --all --install --force

If that doesn't show you any updates, run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/all/.
You should download the Command Line Tools for Xcode 14.1.


Warning: Some taps are not on the default git origin branch and may not receive
updates. If this is a surprise to you, check out the default branch with:
  git -C $(brew --repo gcenx/wine) checkout master

Warning: Your Xcode (14.0.1) is too outdated.
Please update to Xcode 14.1 (or delete it).
Xcode can be updated from the App Store.


Warning: Your Xcode (14.0.1) is outdated.
Please update to Xcode 14.1 (or delete it).
Xcode can be updated from the App Store.
```

### Verification

- [X] I ran `brew update` and am still able to reproduce my issue.
- [ ] ~~I have resolved all warnings from `brew doctor` and that did not fix my problem.~~ I can't solve all warnings.

### What were you trying to do (and why)?

I was going to make an issue on Homebrew/homebrew-core about Ghidra.

### What happened (include all command output)?

When I ran `brew gist-logs ghidra` it showed errors about the Xcode version and CLT version.

```
Error: Your Xcode (14.0.1) is too outdated.
Please update to Xcode 14.1 (or delete it).
Xcode can be updated from the App Store.

Error: Your Command Line Tools (CLT) does not support macOS 13.
It is either outdated or was modified.
Please update your Command Line Tools (CLT) or delete it if no updates are available.
Update them from Software Update in System Preferences or run:
  softwareupdate --all --install --force

If that doesn't show you any updates, run:
  sudo rm -rf /Library/Developer/CommandLineTools
  sudo xcode-select --install

Alternatively, manually download them from:
  https://developer.apple.com/download/all/.
You should download the Command Line Tools for Xcode 14.1.
```

### What did you expect to happen?

Getting logs without any errors, or at least without complaints about the version. Since I already installed the latest version of Xcode.

### Step-by-step reproduction instructions (by running `brew` commands)

```shell
# tried to fix the error
sudo rm -rf /Library/Developer/CommandLineTools
sudo xcode-select --install
# try again
brew gist-logs ghidra
# says that Xcode is outdated
```

**Comment by Bo98** [comment]
This is expected if you are using macOS 13.

The macOS 13 SDK is only available with Xcode 14.1 and unfortunately the final release was delayed.

You can download RC 2 (which should be the same build as the upcoming final release) by following this part of the instructions:

```
Alternatively, manually download them from:
  https://developer.apple.com/download/all/.
```

I reckon the final release will appear today but we shall see.

**Comment by Helloyunho** [comment]
> This is expected if you are using macOS 13.
> 
> The macOS 13 SDK is only available with Xcode 14.1 and unfortunately the final release was delayed.
> 
> You can download RC 2 (which should be the same build as the upcoming final release) by following this part of the instructions:
> 
> ```
> Alternatively, manually download them from:
>   https://developer.apple.com/download/all/.
> ```
> 
> I reckon the final release will appear today but we shall see.

Oh okay, I think I'll just wait for it to be released then.
